recipe/meta.yaml

```
@@ -41,37 +41,32 @@ requirements:
     - jpeg      # [not win]
     - libpng
   host:
-    # GPU requirements
-    - cudatoolkit {{ cudatoolkit }}*  # [pytorch_variant == "gpu"]
-    # other requirements
     - python
-    - jpeg      # [not win]
-    - libpng
     - pip
-    - pillow >=5.3.0,!=8.3.*
-    - pytorch
     - wheel
+    - numpy {{ numpy }}
+    - pytorch 2.0.1
+    # libpng and libjpeg must be available at compilation time
+    - jpeg 9e  # [not win]
+    - libpng 1.6.39
```

```suggestion
    - libpng {{ libpng }}
```
It's in conda_build_config.yaml

---

great, thanks!